server/publications/messages.js

```diff
@@ -37,3 +37,9 @@ Meteor.methods({
 		return Meteor.call('getChannelHistory', { rid, latest: latestDate, oldest: oldestDate, inclusive, count, unreads });
 	},
 });
+
+Meteor.methods({
+	'chatMessage/get'() {
+		return [];
```

Comment:
Why is this empty?

Reply:
Cached Collection expects calls this message as part of loading step, so returning an empty array to make sure it is initialised at startup